libp2p/protocols/rendezvous.nim

```diff
@@ -701,9 +736,13 @@ proc new*(T: typedesc[RendezVous], rng: ref HmacDrbgContext = newRng()): T =
   return rdv
 
 proc new*(
-    T: typedesc[RendezVous], switch: Switch, rng: ref HmacDrbgContext = newRng()
+    T: typedesc[RendezVous],
+    switch: Switch,
+    rng: ref HmacDrbgContext = newRng(),
+    minDuration = MinimumDuration,
+    maxDuration = MaximumDuration,
 ): T =
-  let rdv = T.new(rng)
+  let rdv = T.new(rng, minDuration, maxDuration)
```

Comment:
Good catch from @lchenut. We should check invalid values from min and max. I see that it is possible to create a negative `Duration` instance. Does it make sense? If so, we need to validate it here too.